posthog/models/organization.py

```diff
@@ -79,7 +79,7 @@ class PluginsAccessLevel(models.IntegerChoices):
     updated_at: models.DateTimeField = models.DateTimeField(auto_now=True)
     domain_whitelist: ArrayField = ArrayField(
         models.CharField(max_length=256, blank=False), blank=True, default=list
-    )  # used to allow self-serve account creation based on social login (#5111)
+    )  # Used to allow/block self-serve account creation based on social login (#5111)
```

Comment:
Technically this doesn't block self-serve account creation. If you log in with SSO and your base domain does not match this list you can still use an invite (i.e. is completely disregarded)